lib/idx/remediatiors/Base.ts

```diff
@@ -0,0 +1,86 @@
+/* eslint-disable complexity */
+import { RemediationValues, IdxRemediation, IdxToRemediationValueMap } from '../types';
+import { getAllValues, getRequiredValues } from '../util';
+
+export default class Base {
+  remediation: IdxRemediation;
+  values: RemediationValues;
+  map?: IdxToRemediationValueMap;
+
+  constructor(remediation: IdxRemediation, values: RemediationValues) {
+    this.remediation = remediation;
+    this.values = values;
+  }
+
+  canRemediate() {
+    if (!this.map) {
+      return false;
+    }
+    const required = getRequiredValues(this.remediation);
+    const needed = required.find((key) => {
```

Comment:
`find` -> `some` (<boolean>)

Reply:
I am preferring `find` so we can print the missing key